lib/active_outbox/outboxable.rb

```diff
@@ -10,16 +10,21 @@ module Outboxable
       *namespace, klass = name.underscore.upcase.split('/')
       namespace = namespace.reverse.join('.')
 
-      module_parent.const_set('Events', Module.new) unless module_parent.const_defined?('Events')
+      module_parent.const_set('ActiveOutbox', Module.new) unless module_parent.const_defined?('ActiveOutbox', false)
```

Comment:
what is `module_parent` ? is it configurable?

What I was saying is that any class created by this gem should go in `::ActiveOutbox` namespace

Reply:
that would be the module name of where this concern is included. That we are not creating ourselves

Reply:
hmm ok, I think we are good then